src/app/models/types/curationData.ts

```diff
@@ -45,6 +46,7 @@ export interface CurationProps {
   curationLength?: number;
   mostRead?: MostReadData;
   nthCurationByStyleAndProminence?: number;
+  radioSchedule?: Partial<RadioScheduleData[]>;
```

Comment:
Let's remove `Partial` from here too - the intention is to wrap the entire interface - `CurationProps` in `Partial` so that any mandatory field becomes optional. (and only in test code)
```suggestion
  radioSchedule?: RadioScheduleData[];
```